Makefile

```diff
@@ -254,6 +254,10 @@ gql-gen:
 build-graphql-server: gql-gen
 	@CGO_ENABLED=0 GOOS=linux go build -o bin/graphql-server graphql-server/go-server/main.go
 
+# sdk for graphql-server api
+gql-sdk-upgrade:
+	docker run -v $(shell pwd)/graphql-server/go-server/graph/schema:/schema 172.22.96.34/dev-branch/arcadia-bff-sdk:main
```

Comment:
Perhaps we should push the image  `172.22.96.34/dev-branch/arcadia-bff-sdk:main` to `dockerhub`. @bjwswang @Abirdcfly

Reply:
Taking it a step further, should we make the `make gql-sdk-upgrade` command auto-triggered to automatically run in github action as soon as some files change?

Reply:
1st. Never tag/use image with a internal image registry. Cause we are opensourced project!
2nd. yes. Make this a github action and triggered automatically. The generated sdk should be published to a publish npm registry.

Reply:
1. Using the image `kubebb/gql-sdk-generator:main`
2. Put the Github action on my to do list, I  need to learn how to write the workflow.